### DISCUSSION TITLE
Section 1: "largely by volunteers" in conflict with Fig 3?

### DISCUSSION THREAD

**Discussion by hamogu** [astropy/astropy-v5.0-paper#67] [Text comments] [answered]
> The Project is maintained and coordinated largely by volunteers and contributors who are primarily paid for other reasons (i.e., Astropy work is just a means to an end).

Fig 3 makes "software developer" that largest group today. That seems to be at odds with this statement of "largely by volunteers". I see two ways this could be going: Either rewords here to repeat the point that there is no single institution, i.e. the software developers contribute/vote/direct with the same power that other contributors have, not with a "my institution directs astropy to do X" (in other words, there is little difference in the way that software engineers are treated to the other groups in Fig 3 - neither do the students direct the software developers around, nor do the software developers get to dictate everything) or explain that "largely" is a measure of total people contributing, while Fig 3 is about power-contributors.

**Comment by pllim** [comment]
Might be related: #66

**Comment by adrn** [comment, marked as answer]
I think maybe you misunderstood the point here: The sentence you quote doesn't state that the project is developed "largely by volunteers". It is meant to imply that the project is developed largely by volunteers AND software engineers who work on astropy indirectly. Maybe you would agree more if the order was flipped?

> The Project is maintained and coordinated largely by contributors who are primarily paid for other reasons (i.e., Astropy work is just a means to an end) and by volunteers.

**Reply by adrn** [reply]
Either way, if this sentence is misleading, please open a PR to suggest a new phrasing.

**Reply by hamogu** [reply]
Will do.
